linux_os/guide/services/ntp/service_timesyncd_root_distance_configured/oval/shared.xml

```diff
@@ -0,0 +1,52 @@
+<def-group>
+  <definition class="compliance" id="{{{ rule_id }}}" version="1">
+    {{{ oval_metadata("Ensure that timesyncd RootDistanceMaxSec is configured") }}}
+    <criteria comment="Timesyncd servers are configured" operator="AND">
+      <extend_definition comment="service timesyncd enabled"
+        definition_ref="service_timesyncd_enabled" />
+      <criteria comment="service systemd-timesyncd is configured to start and is running" operator="AND">
+        <criterion comment="systemd-timesyncd is running"
```

Comment:
`service_timesyncd_enabled` via template also ensures the service is running. This criterion should not be necessary here.

Reply:
Dropped those from both rules' ovals. Thanks for the clarification on this topic :bow: